modin/pandas/base.py

```diff
@@ -283,6 +286,8 @@ def _build_repr_df(
             indexer = row_indexer, _get_repr_axis_label_indexer(self.columns, num_cols)
         else:
             indexer = row_indexer
+        if isinstance(self._query_compiler, PlainPandasQueryCompiler):
+            return self._query_compiler.to_pandas().iloc[indexer]
```

Comment:
Why is this change needed? Isn't it sufficient the line below?

Reply:
Removed